be/src/runtime/stream_load/stream_load_pipe.cpp

```diff
@@ -275,6 +277,67 @@ Status StreamLoadPipe::_push_front_unlocked(const ByteBufferPtr& buf) {
     return Status::OK();
 }
 
+CompressedStreamLoadPipeReader::CompressedStreamLoadPipeReader(std::shared_ptr<StreamLoadPipe> pipe,
+                                                               TCompressionType::type compression_type)
+        : StreamLoadPipeReader(std::move(pipe)), _compression_type(compression_type) {}
+
+StatusOr<ByteBufferPtr> CompressedStreamLoadPipeReader::read() {
+    if (_decompressor == nullptr) {
+        auto compression = CompressionUtils::to_compression_pb(_compression_type);
+        if (compression == CompressionTypePB::UNKNOWN_COMPRESSION) {
+            return Status::NotSupported("Unsupported compression algorithm: " + std::to_string(_compression_type));
```

Comment:
std::to_string(_compression_type) convert to the string "LZ4" or "4"?

Reply:
LZ4